services/QuillLMS/client/app/bundles/Connect/components/home.tsx

```diff
@@ -2,18 +2,20 @@ import * as React from 'react';
 import { useQuery } from 'react-query';
 import { connect } from 'react-redux';
 import { renderRoutes } from "react-router-config";
+import { withTranslation } from 'react-i18next';
 
 import { NavBar } from './navbar/navbar';
 
 import { addKeyDownListener } from '../../Shared/hooks/addKeyDownListener';
+import i18n from '../../Shared/libs/translations/i18n';
 import { ScreenreaderInstructions, TeacherPreviewMenu, getlanguageOptions, } from '../../Shared/index';
 import { fetchUserRole } from '../../Shared/utils/userAPIs';
 import { setLanguage } from '../actions.js';
 import { lessonUid } from '../app';
 import { getParameterByName } from '../libs/getParameterByName';
 import { routes } from "../routes";
 
-export const Home = ({playLesson, lessons, dispatch}) => {
+export const Home = ({ playLesson, lessons, dispatch, t }) => {
```

Comment:
It's probably not a big deal, but I prefer more descriptive names for variables.  Maybe just call this `translate` to match what the attribute that we're passing it to?

Reply:
I agree, but this is actually from the library-- I can't change the name. It comes populated from wrapping this component in the `withTranslation` wrapper